include/circt/Dialect/Verif/Passes.td

```diff
@@ -42,4 +42,12 @@ def LowerFormalToHWPass : Pass<"lower-formal-to-hw", "mlir::ModuleOp"> {
   }];
 }
 
+def LowerContractsPass : Pass<"lower-contracts", "mlir::ModuleOp"> {
+  let summary = "Lower contracts into formal tests";
+  let description = [{
+    Converts `hw.module` ops containing a `verif.contract` into a
+    `verif.formal` op.
```

Comment:
Might be nice to have a slightly more detailed explanation (and maybe an example) here (it's essentially already in the rationale, but it might be nice to repeat the most important points here.